.gitignore

```diff
@@ -18,6 +18,3 @@ cmake-build-debug
 *_REMOTE_*.txt
 *.orig
 
-# Symbols database for stuff like autocomplete in Vim
```

Comment:
I'm gonna guess this is intentional?

Reply:
Yeah, we now have a symlink there for the `compile_commands.json` file, so we don't want to ignore it anymore.